packages/core/src/styles/styles.scss

```diff
@@ -0,0 +1,68 @@
+@import "./mixins.scss";
+@import "./components/index.scss";
+@import "./graphs/index.scss";
+@import "./type";
+@import "./chart-holder";
+
+
+$default_transition: all 0.1s ease-out;
+
+
+	svg.chart-svg {
```

Comment:
There's some weird indentation here ... also I'd make sure these classes are namespaced well enough (`#{$prefix}` and all that)